lexers/autoit.lua

```diff
@@ -2,26 +2,59 @@
 -- AutoIt LPeg lexer.
 -- Contributed by Jeff Stone.
 
-local lexer = require('lexer')
-local token, word_match = lexer.token, lexer.word_match
+local lexer = lexer
 local P, S = lpeg.P, lpeg.S
 
-local lex = lexer.new('autoit')
-
--- Whitespace.
-lex:add_rule('whitespace', token(lexer.WHITESPACE, lexer.space^1))
+local lex = lexer.new(...)
 
 -- Keywords.
-lex:add_rule('keyword', token(lexer.KEYWORD, word_match({
+lex:add_rule('keyword', lex:tag(lexer.KEYWORD, lex:word_match(lexer.KEYWORD, true)))
+
+-- Functions.
+lex:add_rule('function_builtin',
+  lex:tag(lexer.FUNCTION_BUILTIN, lex:word_match(lexer.FUNCTION_BUILTIN, true)) *
+    #(lexer.space^0 * P('(')^-1))
+local func = lex:tag(lexer.FUNCTION, lexer.word)
+lex:add_rule('function', S('_')^0 * func * #(lexer.space^0 * '('))
```

Comment:
`S('_')^0` shouldn't be needed because `lexer.word` recognizes '_' as word characters.

Reply:
Good point.